app/graph/state.py

```
@@ -0,0 +1,20 @@
+from typing import Dict, List, Optional, Any
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)